src/hooks/VoiceRecorder/index.tsx

```diff
@@ -62,7 +71,7 @@ export const VoiceRecorderProvider = (props: VoiceRecorderProps): React.ReactEle
         logger.info('VoiceRecorder: Succeeded getting media stream.', stream);
         setIsRecordable(true);
         const mediaRecorder = new MediaRecorder(stream, {
-          mimeType: VOICE_RECORDER_MIME_TYPE,
+          mimeType: browserSupportMimeType,
```

Comment:
Where was this not supoprted?

Reply:
It was Safari

Reply:
Then, we cannot record voice on mobile safari?

Reply:
now we can record in Safari